### DISCUSSION TITLE
Week 12 post

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#218] [General]
<img width="469" alt="Screenshot 2023-07-27 190204" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/65b14fd9-08c5-4dfa-8497-3eac6b4b39e9">

Link to project: https://github.com/USU-ITLS-x265-23-summer/Skye404Project/tree/main

@kensnyder Do I need to focus on other requirements in the rubric? For example, some of the topics that are listed. Or should I be focusing on making the page more functional/complete? I don't want to miss anything, but just getting the website going is challenging in and of itself.

**Comment by CJ3315** [comment]
Wow! Great Progress made this week! This is awesome to see what you have done. What has been your biggest struggle? What kind of improvements are you wanting to make?

**Reply by Skye404** [reply]
I want it all to look professional, not just a basic web page from a beginner like me. haha I'd like to figure out how to embed the images and resume, so they appear as an image and when clicking in it, it then opens the preview page of the document. I want the graphic design images to show up to view too.

**Comment by derekscholes** [comment]
Just a question, do you have a link to the official page published page or only to GitHub repo?

**Reply by Skye404** [reply]
Just the GitHub repo. I've had problems getting it set up within my own personal repo. Sounds like I need to copy and paste my code over to another repo, but I, of course, ran into many issues doing that... so it's still a work in progress. :/

**Comment by kensnyder** [comment]
> @kensnyder Do I need to focus on other requirements in the rubric? For example, some of the topics that are listed. Or should I be focusing on making the page more functional/complete? I don't want to miss anything, but just getting the website going is challenging in and of itself.

I think you mean the [rubric here](https://github.com/USU-ITLS-x265-23-summer/class-repo/blob/main/Weekly-assignments/Projects.md#project-assessment). As a 6265 student, you'll want one more breadth point. You currently have points for HTML, CSS, and git. Some examples on other things you can add:

1. Use JavaScript check to see if the "message" field has at least 20 characters and alert to the user that they need a longer response.
2. Apply some [aria-* attributes](https://developer.mozilla.org/en-US/docs/web/Accessibility/ARIA/Attributes) to your HTML especially your form elements.
3. Deploy your project to GitHub pages and [create a QR code that](https://www.qr-code-generator.com/) links to your project page.
4. Use npm to install prettier and add a command to a package.json file that reformats your code.
5. Add a second HTML page and link to it from the first.

I can help with any of these things.

**Reply by Skye404** [reply]
@kensnyder Thanks! I'll come to office hour Saturday because I've run into some challenges.

**Reply by Skye404** [reply]
![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/e50a68aa-bc76-476e-b1a2-0b139735c4c6)

This QR code should go to my website.

**Comment by audreyhki** [comment]
Great going Skye! You've made substantial progress this semester. A suggestion, if you have not yet created a second HTML page to link to your first page like Professor Synder suggested on point 5, this would be fairly easy to accomplish. You can create an exact copy of your current page, this way the copy will have all the same styles in the header and footer, and then edit the contents of the main and body elements.

**Comment by kassyoveson** [comment]
Hi Sky. You have come a long way in your project. It is looking great. I like the color that you have chosen and the links work, so that's amazing. I know how much headache that gave you over the last few weeks. Great job!

**Reply by Skye404** [reply]
Thanks!

**Comment by beto0890** [comment]
Significant progress. I suggest having a personal deadline or goal to achieve at a certain point.
